pkg/hubble/parser/common/endpoint.go

```diff
@@ -167,19 +168,22 @@ func (r *EndpointResolver) ResolveEndpoint(ip netip.Addr, datapathSecurityIdenti
 		}
 	}
 	var labels []string
+	var clusterName string
 	if r.identityGetter != nil {
 		if id, err := r.identityGetter.GetIdentity(numericIdentity); err != nil {
 			r.log.WithError(err).WithField("identity", numericIdentity).
 				Debug("failed to resolve identity")
 		} else {
 			labels = SortAndFilterLabels(r.log, id.Labels.GetModel(), identity.NumericIdentity(numericIdentity))
+			clusterName = (id.Labels[k8sConst.PolicyLabelCluster]).Value
```

Comment:
we might not have this label for reserved identity (e.g. kube-apiserver, hosts, ingress, health, etc),  hence might need to add an existence check here.

Reply:
`Labels` is [a map of `Label`](https://github.com/cilium/cilium/blob/962e78dfbd41ae80269ccff955f401f6f28d66a5/pkg/labels/labels.go#L166-L167), not `Label` pointers and the default value is then an empty string. The unit tests would panic if this wasn't the case.

Reply:
Ah nice, maybe worth a comment but not blocking.